orc8r/cloud/go/test_utils/test_db.go

```diff
@@ -36,7 +35,7 @@ var (
 // GetSharedMemoryDB returns a singleton in-memory database connection.
 func GetSharedMemoryDB() (*sql.DB, error) {
 	var err error
-	once.Do(func() { instance, err = sqorc.Open(storage2.SQLDriver, ":memory:") })
+	once.Do(func() { instance, err = sqorc.Open("sqlite3", ":memory:") })
```

Comment:
I think there's a const for this somewhere in the codebase. If not, let's add one -- I think there's one for postgres you could look for to emulate

Reply:
couldn't find the postgres const so I moved it to the definitions const file